pkg/kubectl/history.go

```diff
@@ -111,7 +113,12 @@ func (h *DeploymentHistoryViewer) ViewHistory(namespace, name string, revision i
 		if !ok {
 			return "", fmt.Errorf("unable to find the specified revision")
 		}
-		return printTemplate(template)
+
+		templateString, err := printTemplate(template)
+		if err != nil {
+			return "", err
+		}
+		return "Revision Time:\t" + revisionTime[revision] + "\n" + templateString, nil
```

Comment:
Would you need some protection for existing deployments that currently doesn't have the annotation?